### PR TITLE
Replace /var with localstatedir placeholder in systemd unit file

### PR DESCRIPTION
**Pull request by tu-maurice**
In certain environments one cannot assume the localstatedir to be /var. Gentoo e.g. uses /var/lib. I don't know if this is even the right fix or if localstatedir might be misused for files that are supposed to got into /var/log.

### CODE REVIEW

**Comment by tu-maurice** [comment]
^ forgot to replace localstatedir in usbguard.service

File: Makefile.am [hunk 1]
```diff
@@ -115,7 +115,11 @@ CLEANFILES+=\
 	$(top_builddir)/usbguard.service
 
 usbguard.service: $(top_srcdir)/usbguard.service.in
-	$(SED) -e "s|%sbindir%|${sbindir}|" -e "s|%sysconfdir%|${sysconfdir}|" $^ > $@ || rm -f $@
+	$(SED) \
+	  -e "s|%sbindir%|${sbindir}|" \
+	  -e "s|%sysconfdir%|${sysconfdir}|" \
+	  -e 's|%localstatedir%|${localstatedir}|" \
```
Comment (radosroka) [inline Makefile.am]: Could you replace the first ```'``` wtih ```"``` otherwise it is not valid.

Comment (tu-maurice) [reply Makefile.am]: Oh, sure. My bad.

**Comment by radosroka** [comment]
Thanks.

**Pull request merged**